macros/detect_baron_samedit_cve_2021_3156_filter.yml

```diff
@@ -0,0 +1,3 @@
+definition: search *
```

Comment:
we dont need this file, the backend tool will create this automatically. You can delete this here